bdk_electrum_example/src/electrum.rs

```diff
@@ -182,10 +312,10 @@ impl ElectrumClient {
                     for (txid, pos) in txid_list {
                         if let Err(failure) = sparse_chain.insert_tx(txid, pos) {
                             match failure {
-                                sparse_chain::InsertTxFailure::TxTooHigh { .. } => {
+                                InsertTxFailure::TxTooHigh { .. } => {
                                     unreachable!("We should not encounter this error as we ensured tx_height <= tip.height");
                                 }
-                                sparse_chain::InsertTxFailure::TxMovedUnexpectedly { .. } => {
+                                InsertTxFailure::TxMovedUnexpectedly { .. } => {
```

Comment:
nit: I prefer to keep the `sparse_chain::` qualification so I know which module it's from.